### ISSUE TITLE
[0.9.3] (Bug) Song restarts when modifying progress bar during local playback

### ISSUE THREAD

**Issue by jasperaelvoet** [jmshrv/finamp#651]
When using local playback, modifying the progress of the song will restart it from the beginning.

This might not be related, but local (transcoded from FLAC) AAC playback is not working (does not start playing), even tho Jellyfin is on 10.9.0.

Device: iPhone SE3
iOS version: 17.4.1
Jellyfin version: 10.9.0 (daa56f6c78f6) from linuxserver.io (issue also occurred on latest stable release)

Reproducing the issue:
https://github.com/jmshrv/finamp/assets/82276344/24dc1f01-0774-470b-8bbb-9a7574c01606

**Comment by Komodo5197** [comment]
Some analysis of the downloaded transcode file vs the original on the server suggests mp3 may suffer from the same sort of header writeback issue as caf/aac, just less bad.

$ diff -y --suppress-common-lines <(xxd transcodes/67b6d13e83f86d281defcf898903b2ed.mp3) <(xxd songs/2
967c2fa4746a26c9fbac883e8d75a57.mp3)
00000190: 496e 666f 0000 000f 0000 0d6d 0015 ed0e  Info...... | 00000190: 5869 6e67 0000 000f 0000 0000 0000 0000  Xing......
000001a0: 0003 0608 0a0d 0f11 1517 1a1c 1f21 2327  .......... | 000001a0: 0002 0507 0a0c 0f11 1416 191c 1e21 2326  ..........
000001b0: 292c 2e30 3335 393b 3e40 4245 474b 4d4f  ),.0359;>@ | 000001b0: 282b 2d30 3335 383a 3d3f 4244 4749 4c4f  (+-0358:=?
000001c0: 5254 5759 5d5f 6164 6668 6b6e 7173 7678  RTWY]_adfh | 000001c0: 5154 5659 5b5e 6063 6668 6b6d 7072 7577  QTVY[^`cfh
000001d0: 7a7d 8083 8587 8a8c 8f91 9497 999c 9ea0  z}........ | 000001d0: 7a7c 7f82 8487 898c 8e91 9396 999b 9ea0  z|........
000001e0: a3a6 a9ab aeb0 b2b5 b8bb bdbf c2c4 c7ca  .......... | 000001e0: a3a5 a8aa adaf b2b5 b7ba bcbf c1c4 c6c9  ..........
000001f0: cccf d1d4 d6d8 dcde e1e3 e6e8 eaee f0f3  .......... | 000001f0: ccce d1d3 d6d8 dbdd e0e2 e5e8 eaed eff2  ..........
00000200: f5f7 fafc 0000 0000 4c61 7663 3539 2e33  ........La | 00000200: f4f7 f9fc 0000 0000 4c61 7663 3539 2e33  ........La
00000210: 3700 0000 0000 0000 0000 0000 0024 0676  7......... | 00000210: 3700 0000 0000 0000 0000 0000 0000 0000  7.........
00000220: 0000 0000 0015 ed0e cdd3 525d 0000 0000  .......... | 00000220: 0000 0000 0000 0000 0000 0000 0000 0000  ..........

It seems like most other players can just ignore the issue and seek normally, but iOS is finicky.

An analysis of an AAC download suggests something similar but worse.  10.9 AAC aren't completely busted like 10.8 ones, but Android and windows can't scrub through them.  Theoretically these were validated to match the server versions with the 10.9 patches, so I'm not sure what happened here.  I'll try to investigate more.

**Comment by Chaphasilor** [comment]
@jasperaelvoet this should be fixed now, can you confirm?

**Comment by Chaphasilor** [comment]
yeah, it seems like there's currently nothing we can do about this. It's a bug in the server/ffmpeg that causes issues on iOS
